lumicks/pylake/tests/test_imaging_confocal/conftest.py

```diff
@@ -0,0 +1,211 @@
+import pytest
+import numpy as np
+from lumicks.pylake.point_scan import PointScan
+from lumicks.pylake.kymo import Kymo
+from lumicks.pylake.scan import Scan
+from ..data.mock_file import MockDataFile_v2
+from ..data.mock_confocal import MockConfocalFile, generate_scan_json
+
+
+start = np.int64(20e9)
+dt = np.int64(62.5e6)
+
+# fmt: off
+@pytest.fixture(scope="module")
+def reference_counts():
+    return np.uint32([2, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 2, 1, 8, 0,
+                      0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 8, 0,
+                      1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 8, 0,
+                      0, 0, 0, 0, 1, 0, 0, 1, 0, 0, 1, 0, 0, 0, 8, 0])
+
+reference_infowave = np.uint8([1, 0, 2, 1, 0, 2, 1, 0, 2, 1, 0, 2, 1, 0, 0, 2,
+                               0, 1, 2, 0, 1, 2, 0, 1, 2, 0, 1, 2, 0, 1, 0, 2,
+                               1, 0, 2, 1, 0, 2, 1, 0, 2, 1, 0, 2, 1, 0, 0, 2,
+                               0, 1, 2, 0, 1, 2, 0, 1, 2, 0, 1, 2, 0, 1, 0, 2])
+# fmt: on
+
+
+@pytest.fixture(scope="module")
+def reference_timestamps():
+    stop = start + len(reference_infowave) * dt
+    return np.arange(start, stop, 6.25e7).astype(np.int64)
+
+
+@pytest.fixture(scope="module")
+def test_point_scans(reference_counts):
+    point_scans = {}
+
+    mock_file, metadata, stop = MockConfocalFile.from_streams(
+        start,
+        dt,
+        [],
+        [],
+        [],
+        infowave=reference_infowave,
+        red_photon_counts=reference_counts,
+        green_photon_counts=reference_counts,
+        blue_photon_counts=reference_counts,
+    )
+    point_scans["PointScan1"] = PointScan("PointScan1", mock_file, start, stop, metadata)
+
+    return point_scans
+
+
+@pytest.fixture(scope="module")
+def test_kymos(reference_counts):
+    kymos = {}
+
+    mock_file, metadata, stop = MockConfocalFile.from_streams(
+        start,
+        dt,
+        [0],
+        [5],
+        [10.0],
+        infowave=reference_infowave,
+        red_photon_counts=reference_counts,
+        green_photon_counts=reference_counts,
+        blue_photon_counts=reference_counts,
+    )
+    kymos["Kymo1"] = Kymo("Kymo1", mock_file, start, stop, metadata)
+
+    mock_file, metadata, stop = MockConfocalFile.from_streams(
+        start,
+        dt,
+        [0],
+        [5],
+        [10.0],
+        infowave=reference_infowave,
+        red_photon_counts=reference_counts,
+        green_photon_counts=reference_counts,
+        blue_photon_counts=reference_counts,
+    )
+    kymos["damaged"] = Kymo("fast Y slow X", mock_file, start - 62500000, stop, metadata)
```

Comment:
Key name could indicate what's broken about it. Or indeed as Dean mentioned, a separate fixture with a docstring.

Also the name of the `Kymo` itself is not consistent with the name of the key which could be confusing if it fires?

Reply:
good catch, copy/paste error